cli/src/cortx_prep.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+set -euE
+
+LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/cortx_prep.log}"
+if [[ ! -e "$LOG_FILE" ]]; then
+    mkdir -p $(dirname "${LOG_FILE}")
+    touch "${LOG_FILE}"
+fi
+
+function trap_handler {
+    echo -e "\n***** FAILED!!*****" 2>&1 | tee -a $LOG_FILE
+    echo "Detailed error log is kept at: $LOG_FILE" 2>&1 | tee -a $LOG_FILE
+    exit 1
+}
+trap trap_handler ERR
+
+function install_prvsnr() {
+
+    echo "INFO: Creating a directory to serve as the mount point" 2>&1 | tee -a ${LOG_FILE}
+    mkdir -p /tmp/iso_mount
+
+    if [[ `find /root/iso -name *.iso` ]]; then
+        cortx_iso=$(ls -t /root/iso/cortx-1.0-*-single.iso | head -1 | xargs basename)
```

Comment:
make this a bit more universal, e.g. `cortx-*.iso`. this will help us in the future.